insight-engine/latest/install/options.md

```diff
@@ -2,7 +2,9 @@
 title: Installation options
 ---
 
-You can install Search and Insight Engine using the distribution zip or Docker Compose. There are three different ways you can install the software, the first is with mutual TLS, the second is without mutual TLS (plain HTTP), and the third is without mutual TLS (HTTP with secret word in request header).
+You can install Search and Insight Engine using the distribution zip or Docker Compose. There are two different ways you can install the software, the first is with mutual TLS, and the second is without mutual TLS (HTTP with secret word in request header).
+
+**Important:** From version 2.0 you cannot use the without mutual TLS (plain HTTP) installation method because it is no longer supported.
```

Comment:
```suggestion
> **Important:** From version 2.0, you cannot install Search and Insight Engine without mutual TLS (plain HTTP) because it is no longer supported.
```